cypress/integration/finance/funds/funds.search.spec.js

```diff
@@ -1,136 +1,101 @@
 import uuid from 'uuid';
-
-import {
-  Accordion,
-  Button,
-  Checkbox,
-  MultiColumnList,
-  MultiSelect,
-  SearchField,
-  Selection,
-  SelectionList,
-} from '../../../../interactors';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import TopMenu from '../../../support/fragments/topMenu';
+import Funds from '../../../support/fragments/finance/funds/funds';
+import FinanceHelp from '../../../support/fragments/finance/financeHelper';
+import { MultiColumnList } from '../../../../interactors';
 import { testType } from '../../../support/utils/tagTools';
 
-// TODO: refactoring needed in order to have 1-1 relation between smoke tests in Test Rail and TAF
 describe('ui-finance: Funds list search and filter', () => {
-  const timestamp = (new Date()).getTime();
+  let aUnits;
+  let tags;
+  let ledgers;
+  let groups;
+  let fundTypes;
```

Comment:
May be better to create object? Then you can fill object with values inside code.
```
objectName = {}
```
And the you can create values like this:
```
cy.getAcqUnitsApi({ limit: 1 })
      .then(({ body }) => {
        objectName.aUnits = body.acquisitionsUnits;
      });
```
Just my suggestion.